toolkit/pythonit_toolkit/api/client.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Any, Dict, Optional
+
+import httpx
+import jwt
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+
+INTERNAL_JWT_SECRET = os.getenv("INTERNAL_JWT_SECRET", "a-very-secret-secret")
+
+
+class Client:
+    url = "/internal-api"
+
+    async def execute(
+        self,
+        query: str,
+        variables: Optional[Dict[str, Any]] = None,
+    ):
+        token = jwt.encode({"some": "payload"}, INTERNAL_JWT_SECRET, algorithm="HS256")
+
+        async with httpx.AsyncClient() as client:
+            response = await client.post(
+                self.url,
```

Comment:
uhm how does it work? why only `/internal-api`?

Reply:
We need it only for the internal-api , right?

Reply:
The internal api is an api that can exist on any service, so it needs to have the url of the service you are calling

Reply:
So association backend might call pycon backend

pycon backend might call users backend

and so on

Reply:
So I add the url in the init method